src/Http/Http.Results/test/ProblemResultTests.cs

```diff
@@ -64,11 +64,43 @@ public async Task ExecuteAsync_UsesDefaults_ForValidationProblemDetails()
         Assert.Equal(StatusCodes.Status400BadRequest, httpContext.Response.StatusCode);
         stream.Position = 0;
         var responseDetails = JsonSerializer.Deserialize<HttpValidationProblemDetails>(stream);
-        Assert.Equal("https://tools.ietf.org/html/rfc7231#section-6.5.1", responseDetails.Type);
+        Assert.Equal("https://tools.ietf.org/html/rfc9110#section-15.5.1", responseDetails.Type);
         Assert.Equal("One or more validation errors occurred.", responseDetails.Title);
         Assert.Equal(StatusCodes.Status400BadRequest, responseDetails.Status);
     }
 
+    [Fact]
+    public async Task ExecuteAsync_UsesDefaults_WhenHttpStatusCodeSet()
```

Comment:
Maybe it could be something like.

```suggestion
    public async Task ExecuteAsync_SetsTitleFromReasonPhrases_WhenNotInDefaults()
```